pkgs/development/tools/ansible-lint/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, pythonPackages, fetchurl, ansible }:
+
+pythonPackages.buildPythonPackage rec {
+  name = "ansible-lint-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.